libevmasm/Assembly.cpp

```diff
@@ -390,7 +390,7 @@ Assembly& Assembly::optimise(OptimiserSettings const& _settings)
 
 map<u256, u256> Assembly::optimiseInternal(
 	OptimiserSettings const& _settings,
-	std::set<size_t> const& _tagsReferencedFromOutside
+	std::set<size_t> _tagsReferencedFromOutside
```

Comment:
Shouldn't it be `&`?

Reply:
No, it is not needed in the caller. The function returns the replacements themselves and then the caller takes them into account.